election.html

```diff
@@ -0,0 +1,289 @@
+<!doctype html>
+<html lang="en_NZ">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Vote with IRMA</title>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+    <link type="text/css" rel="stylesheet" href="style-election.css">
+
+    <!-- development version, includes helpful console warnings -->
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+
+    <script src="irma.js" defer></script>
+    <script type="text/javascript">
+        const AWS_IRMA_SERVER = 'http://13.54.7.159:8088';
+        const IRMA_SERVER = AWS_IRMA_SERVER;
+        const AUTH_METHOD_TOKEN = 'token';
+        const PRIVATE_TOKEN = 'secret-fake-token';
+
+        const MEMBERSHIP_ATTRIBUTES = [
+            'inz-demo.internetnz.membership.id',
+            'inz-demo.internetnz.membership.validUntil'
```

Comment:
What do you think of addin `purchasedAt` too? There is some "you are too young to vote" criteria. We could use that later and use some days as example (1 day, 5 days young...).

Reply:
I added that but removed it because though might not be a good condition to check in our case.